### ISSUE TITLE
[BUG] Example for name in RTD does not comply with regex for "name" 

### ISSUE THREAD

**Issue by tabedzki** [funkyfuture/deck-chores#146]
## Bug

If one uses the example (copied below) for labels from the [deck-chores website](https://deck-chores.readthedocs.io/en/stable/usage.html#containers-without-an-enduring-main-process), one ends up with an error related to the `_` in the example (see below).

Reproduced Code:
```yaml
services:
  neverending:
    # …
    command: tail -f /dev/null
    labels:
      deck-chores.daily_job.command: daily_command …
      deck-chores.daily_job.interval: daily
```

Error:
```
2024-10-12 09:30:51,064|ERROR   |Misconfigured job definition: {'command': 'sh -c "echo \'Hello\' " &> /proc/1/fd/1', 'interval': 'daily', 'name': 'daily_job', 'user': ''}
2024-10-12 09:30:51,064|ERROR   |Errors: {'name': ["value does not match regex '[a-z0-9-]+'"]}
```

## Possible solutions
1. Update the [name regex](https://github.com/funkyfuture/deck-chores/blob/2f48370b444ea5ba24eb29421f367db63ece35c0/deck_chores/config.py#L73) (`[a-z0-9-]+`)  to account for a `_` as provided in the example
2. Update the example be updated to use `-` instead

Which one do you think is a better fix? I personally like idea 1 of making the regex more robust.

**Comment by funkyfuture** [comment]
thanks for reporting, i would have thought somebody had pointed to this in the past, but i can't find anything related.

i'm pretty sure that i "designed" it only for using dashes in order to have *one way to do it* and avoid confusions whether one uses one or the other. obviously that failed.

the targeted clarity is mainly useful when it comes to overriding definitions from a (basing) image. within the definitions itself the used separator is readable in a neighboring line.

still, i tend to option 2.

**Comment by tabedzki** [comment]
Okay sounds good. If that's the case, then let's update those two lines; those are the only ones that I saw on the documentation. 

I also think it would be instructive to have a complete `compose.yaml` with that people could use for a test and then modify for their own needs. Would you welcome a PR that includes a full one?

**Comment by tabedzki** [comment]
So I did some digging and the website itself still uses the `_` for `deck-chores.daily_job.command` but the `usage.rst` does not. 

Would you be able to push the changes to the RTD to reflect your already committed changes after merging #147?

**Comment by funkyfuture** [comment]
Solved with #147 and issued subsequent #148.
